data_generators/BUILD.bazel

```diff
@@ -12,16 +12,14 @@ genrule(
 cat <<EOS > $(OUTS)
 set -euo pipefail
 # xcode-locator prints json with keys as version names and values as paths to the developer dir for that xcode version
-readonly output=\\$$($(location @bazel_tools//tools/osx:xcode-locator) 2>/dev/null | grep : | sed -Ee s/.*'"(.+)": "(.+)".*'/'\\1.0|\\2'/g | sort --version-sort -k 1 -t '|' | tail -n 1)
-readonly xcode_version=\\$${output%%.0|*}
-readonly developer_dir=\\$${output#*|}
+readonly xcode_version=\\$$(xcodebuild -version | head -n1 | sed "s/Xcode //")
```

Comment:
Ideally this these lines / script would run automatically post analysis phase for a given version than be hardcoded to one in rules_ios.. This seems an OK given the current state of it.

Reply:
Yeah, it could be more robust, but it's also just for generating this file to be committed here. If it were running as a repo rule and/or generating for multiple versions, it would be worth building some more infrastructure around it.